static/src/js/util/accessMethods/__tests__/buildAccessMethods.test.js

```diff
@@ -182,42 +187,38 @@ describe('buildAccessMethods', () => {
             ]
           }],
           variables: {
```

Comment:
I'm a bit confused why did the variables in the response need to go into the service associated variables?